### PR TITLE
[AN-4378] Adds highlighting of message that is being edited

### PR DESCRIPTION
**Pull request by shumengye**
#### Description
Adds highlighting (48% opacity in text color) of message that is being edited.

#### Ticket
https://wearezeta.atlassian.net/browse/AN-4378
#### APK

[Download build #7401](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7401/artifact/build/artifact/wire-dev-PR61-7401.apk)
[Download build #7402](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7402/artifact/build/artifact/wire-dev-PR61-7402.apk)

### CODE REVIEW

**Comment by mykola-mokhnach** [comment]
1. Open a conversation
2. Long tap any of your text messages
3. Select Edit
4. Swipe back to conversations list
5. Open the same conversation again

AR:
Edit control is visible, but highlighting is reset

ER:
Highlighting should be preserved after STR
OR
Edit control should be reset

**Comment by shumengye** [comment]
@mykola-mokhnach 
conversation list swipe issue fixed in next build

**Comment by mykola-mokhnach** [comment]
GIF button is present for empty input field if I cancel message edit by tapping hardware Back button:
screen

File: wire-core/src/main/res/values/floats.xml [hunk 1]
```diff
@@ -32,6 +32,7 @@
     <item name="content__progress__touch_scaling" type="dimen" format="float">2</item>
     <item name="content__system_message__user_name__alpha" format="float" type="dimen">0.64</item>
     <item name="content__youtube__alpha_overlay" type="dimen" format="float">0.48</item>
+    <item name="content__message_being_edited__opacitu" format="float" type="dimen">0.48</item>
```
Comment (mykola-mokhnach) [inline wire-core/src/main/res/values/floats.xml]: just a small typo here "opacitu"

**Comment by hakonbo** [comment]
@Yorgos-V ❤️  it!

**Pull request merged**